pkg/k8shandler/status.go

```diff
@@ -149,7 +149,7 @@ func getElasticsearchStatus(namespace string) ([]v1alpha1.ElasticsearchStatus, e
 
 			nodeStatus := v1alpha1.ElasticsearchStatus{
 				ClusterName:   node.Name,
-				Replicas:      node.Spec.Nodes[0].Replicas,
+				Replicas:      node.Spec.Nodes[0].NodeCount,
```

Comment:
nit.  this should probably be 'NodeCount' or a more clear status for when we actually support alternte deployment types would be some form of `NodeCount: {master: 1, data: 1, client:1}'.